internal/app/singularity/pull.go

```diff
@@ -343,12 +343,18 @@ func OciPull(name, imageURI, tmpDir string, ociAuth *ocitypes.DockerAuthConfig,
 	return nil
 }
 
-func convertDockerToSIF(image, cachedImgPath, tmpDir string, noHTTPS bool, authConf *ocitypes.DockerAuthConfig) error {
+func convertDockerToSIF(imgCache *cache.Handle, image, cachedImgPath, tmpDir string, noHTTPS bool, authConf *ocitypes.DockerAuthConfig) error {
+	if imgCache == nil {
+		fmt.Println("Image cache is undefined")
+		os.Exit(42)
```

Comment:
What's the significance of the `42` exit code? I would think this should return an error instead

Reply:
None, i use 42 during debugging and simply forgot to remove it. Sorry about that.